decidim-process-extended/app/decorators/decidim/admin/participatory_space/publish_decorator.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module Decidim::Admin::ParticipatorySpace::PublishDecorator
+  def self.decorate
+    Decidim::Admin::ParticipatorySpace::Publish.class_eval do
+      def call
+        return broadcast(:invalid) if participatory_space.nil? || participatory_space.published?
+
+        Decidim.traceability.perform_action!(:publish, participatory_space, user, **default_options) do
+          participatory_space.publish!
+          # process-extended customization
+          notify_admins
+          # process-extended customization
+        end
+
+        broadcast(:ok, participatory_space)
+      end
+
+      private
```

Comment:
Why don't we follow the same as in other decorators?
Also, I'm not sure that the notification must be inside the traceability block.
```suggestion
    Decidim::Admin::ParticipatorySpace::Publish.class_eval do
      alias_method :original_call, :call

      def call
        subscribe(self) # from whisper gem
        result= original_call
        on(:ok) { notify_admins }
      end
...
```

Reply:
this code not works 😅, I'll investigate how use whisper gem..

Reply:
fixed